### DISCUSSION TITLE
"FollowGame" is it working?

### DISCUSSION THREAD

**Discussion by nobountyhunter** [chr233/ASFEnhance#49] [提问求助 / Support]
does this feature work? I wrote a command and it said something like "done or success," but the game didn't add any subscribers(followers).

**Comment by chr233** [comment]
so can you show me how you use this command, and what appid do you specified
I test this command and it seems work fine.

**Reply by nobountyhunter** [reply]
I use my own game for the test. use the command !folllowgame asf appid, Then I get "<botname> ID: appidmygame | Status: Success .

I logged into my account from my computer and checked, yes indeed I do follow the game, but on steadb my game has not changed the number of subscribers after three days, I have subscribed from 300 accounts.

**p.s I figured out what the problem is, the thing is that when we "follow" through your plugin and asf, it just clicks the "follow" button. But if we "follow" in life through his computer,it automatically subscribes to the game's community group.**

Previously, you could immediately go to the game group using a url link, but now for some reason it just goes to the "game hub"
https://steamcommunity.com/games/APPIDGAME


https://i.imgur.com/No5F0OW.jpg
https://i.imgur.com/oB3Nxvy.jpg

**Reply by chr233** [reply]
> I use my own game for the test. use the command !folllowgame asf appid, Then I get " ID: appidmygame | Status: Success .
> 
> I logged into my account from my computer and checked, yes indeed I do follow the game, but on steadb my game has not changed the number of subscribers after three days, I have subscribed from 300 accounts.
> 
> **p.s I figured out what the problem is, the thing is that when we "follow" through your plugin and asf, it just clicks the "follow" button. But if we "follow" in life through his computer,it automatically subscribes to the game's community group.**
> 
> Previously, you could immediately go to the game group using a url link, but now for some reason it just goes to the "game hub" https://steamcommunity.com/games/APPIDGAME
> 
> https://i.imgur.com/No5F0OW.jpg https://i.imgur.com/oB3Nxvy.jpg

![image](https://user-images.githubusercontent.com/13798576/206853123-5697fc0e-e4c9-4522-9200-3c3c0d9de444.png)
this command do as the same as click this "follow" button

**Comment by nobountyhunter** [comment]
Yes, I understand, but why when I do it through your plugin and asf, the number of "followers" does not change. But if I do it manually through the computer, the number of "follower" changes.

This is most likely due to the fact that your plugin does not join the game group after you click "Follow" . But if we do it by hand through the computer, the number of followers changes

AMOUT MEMBERS IN THIS GROUP = FOLLOW(click button)
https://i.imgur.com/UuA1ohZ.png
https://i.imgur.com/sbmtvqt.png

**Comment by egorcreep** [comment]
After pressing "Follow" button, bots should join in the community group of this game, but apparently they are not doing this if we are doing it through this plugin. But when I click the "follow" button manually from Steam client, account is joining to the group

**Reply by nobountyhunter** [reply]
yeah, https://partner.steamgames.com/doc/marketing/followers

**Comment by nobountyhunter** [comment]
UPD 
I looked at the group members of the game and saw that the bots joined, but I do not understand why the number of "follow" is not updated on the steadb, those who joined through asf and your plugin

**Reply by chr233** [reply]
steamdb's data is not in real time, you can login in the robot's account and check if it works.

I tested myself and it works as expected

**Reply by nobountyhunter** [reply]
> steamdb's data is not in real time, you can login in the robot's account and check if it works.
> 
> I tested myself and it works as expected

Hmmm...I've just been waiting 3-5 days, my followers still haven't updated . 
As far as I know, steamdb updates them once a day. 
I'll see what happens next.

**Reply by chr233** [reply]
did you login in the robot's account and check if followed or not

**Reply by nobountyhunter** [reply]
> did you login in the robot's account and check if followed or not

I was not allowed to enter, on steamdb not allowed to enter if you have not funded your account staem for $5, perhaps because of this, he does not track my bots (followers), as accounts are empty and not unlocked, I have not funded them for $5

**Reply by chr233** [reply]
just check the steam, if the bot followed the game and the game's group.

I think the reason is the account is limited, that's none of my business.

**Reply by chr233** [reply]
by the way, your action is likely to be a kind of abuse of this plugin
